core/chains/evm/config/toml/defaults/Gnosis_Chiado.toml

```diff
@@ -0,0 +1,15 @@
+ChainID = '10200'
+# Gnoisis Finality is approx 8 minutes @ 12 blocks per minute, so 96 blocks
+FinalityDepth = 100
+ChainType = 'xdai'
+LinkContractAddress = '0x57Cb1581A672Dfd9D34f12086B0495aD210FB98f'
+LogPollInterval = '5s'
+
+[GasEstimator]
+EIP1559DynamicFees = true
+PriceDefault = '1 gwei'
+PriceMax = '500 gwei'
+# 1 Gwei is the minimum accepted by the validators (unless whitelisted)
+PriceMin = '1 gwei'
```

Comment:
nit: I think we can further reduce this minimum. I see in the mainnet it goes way less than 1gwei, and we don't have enough data in the testnet to check this?

Reply:
PriceMin and PriceDefault have been removed since they are for legacy txs, which is why I did not see any warnings about this. Thanks